challenge.md

```diff
@@ -6,19 +6,29 @@ Write your answers inside this file, where it's indicated by the comments.
 
 1. We just forked a repo on GitHub.com and want to start working on it locally. What command do we use to do that?
 
+git clone <ssh url>
+
 2. OK, we just wrote some code. What command could we use to see a list of all the changes that have been made since the last commit?
 
+git log
```

Comment:
Sorry, this one should be `git status` - that command will show you a list of all files that have been created/changed since the previous commit. `git diff` is also an acceptable answer, since that command will give you a list of all changes made to files that are _already being tracked_.